web-platform/website/src/components/pages/login/LoginDataContext.tsx

```diff
@@ -34,7 +34,7 @@ export function LoginStateProvider(props: { children: React.ReactNode }) {
   useEffect(() => {
     // If user is present but not verified we set initial state to login
     // since login view displays user not verified error.
-    if (isLoaded && user && !isUserVerified) {
+    if (isLoaded && user && !isUserVerified && !LoginState.EmailAccountCreated) {
```

Comment:
This is just an enum value and will always be false I believe (! of a defined value). We need to extract the login state via useLoginState I believe and then compare.

Reply:
Yes, I meant to do ``` loginState !== LoginState.SignupWithEmail```